src/main/java/org/cascadebot/cascadebot/commands/developer/EvalCommand.java

```diff
@@ -92,7 +92,7 @@ public void onCommand(Member sender, CommandContext context) {
                 String codeToRun = imports + " " + code;
                 String results = String.valueOf(scriptEngine.eval(codeToRun));
                 if (results.isBlank()) results = "Empty result!";
-                PasteUtils.pasteIfLong(results, 2048, context::reply);
+                PasteUtils.pasteIfLong(results, 1950, context::reply);
```

Comment:
arbitrary number?

Reply:
Probably should be `2000` which is the new upper limit

Reply:
I'll add a comment here to explain why (As discussed)